decidim/decidim
# Allow creation of hybrid meetings

#### :tophat: What? Why?
Theses days in-person meetings tend to have an online dimension has they tend to be livestreamed.

#### :pushpin: Related Issues
*Link your PR to an issue*
- https://meta.decidim.org/processes/roadmap/f/122/proposals/15798
- Fixes #?

#### Testing
1. Login to the system 
2. Visit the admin panel, and attempt to create a meeting 
3. On the Meeting type drop down, you will see "Both" which represent the new hybrid event 
4. fill in the data, and save 
5. Pepeat the steps for user created meeting 
6. go to the meetings list in the frontend, then select "both", you should see only the 2 meetings created by you. 

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
